static/js/stream_settings_ui.js

```diff
@@ -48,13 +58,7 @@ export const show_subs_pane = {
     settings(stream_name, invite_only, is_web_public) {
         $(".settings, #stream-creation").hide();
         $(".settings").show();
-        $("#subscription_overlay .stream-info-title").html(
-            render_selected_stream_title({
-                stream_name,
-                invite_only,
-                is_web_public,
-            }),
-        );
+        set_right_panel_title(stream_name, invite_only, is_web_public);
```

Comment:
This works, but it seems like it'd be cleaner to pass this function the `sub` object; I guess we'd need to change that in the caller too, but I don't see a clear reason to be passing fields from the `sub` object rather than the whole object in the code path.